coalib/bearlib/abstractions/Linter.py

```diff
@@ -207,6 +207,9 @@ def check_prerequisites(cls):
             :return:
                 True if operational, otherwise a string containing more info.
             """
+            if not super(LinterBase, cls).check_prerequisites():
+                return False
+
```

Comment:
@userzimmermann I thought returning super.check_prerequisite () would give more information to what check failed. If that's not needed, forget about it :)